app/views/auctions/_bid_button.html.erb

```diff
@@ -1 +1,3 @@
-<a href="<%= new_auction_bid_path(auction) %>" class='usa-button usa-button-outline-inverse button-bid'>Bid <icon class="fa fa-angle-double-right"></icon></a>
+<%= link_to 'Bid <icon class="fa fa-angle-double-right"></icon>'.html_safe,
```

Comment:
I pretty much always use `link_to` -- adding classes doesn't seem too crazy. nor does the icon.  just did a quick SO post and it seems like this is the recommended way: http://stackoverflow.com/a/22358718/1019369

what about this feels edge casey?